src/main/java/appeng/hooks/ticking/TickHandler.java

```diff
@@ -333,6 +333,8 @@ public void registerCraftingSimulation(final World world, final CraftingJob craf
 
     /**
      * Simulates the current crafting requests before they user can submit them to be processed.
+     *
+     * @param world
```

Comment:
I think we're just turning in circles .I remove them, you add them back :-D

Reply:
Rebase with conflicts ftw.